D1673/P1673.bs

```diff
@@ -4726,7 +4747,7 @@ public:
 };
 ```
 
-* *Requires:* `ScalingFactor` and `Reference` shall be *Cpp17CopyConstructible*.
+* *Preconditions:* `ScalingFactor` and `Reference` shall be *Cpp17CopyConstructible*.
```

Comment:
I think the way you write this is "if ScalingFactor or Refererence are not Cpp17CopyConstructible the program is ill-formed." Or?